pkg/build/node.go

```diff
@@ -86,3 +86,18 @@ ENTRYPOINT ["node", "{{ .Main }}"]
 
 	return buf.String(), nil
 }
+
+// expandVersion returns a pinned minor version of Node to use
+func expandVersion(version string) string {
+	switch version {
+	case "15":
```

Comment:
Maybe the default version should live here?

```suggestion
	switch version {
	case "":
	  fallthrough
	case "15":
```